code/frontend/src/app/features/seeker-stats/seeker-stats.component.scss

```
@@ -1,14 +1,17 @@
+  @include page-section-stagger;
+
   > app-tabs {
-    animation: slide-up var(--duration-normal) var(--ease-default) both;
-    animation-delay: 0ms;
     display: block;
-    @include sticky-page-header;
   }
 
   > .tab-content {
     margin-top: var(--space-6);
+    // Chromium retains the compositor layer from slide-up's compositing properties,
```

Stylelint (17.7.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)